src/BulletHurtPlayerEC.cpp

```diff
@@ -25,7 +25,10 @@ void BulletHurtPlayerEC::checkEvent() {
         int damage = bullet->getDamage();
         LifeC* playerHealth = dynamic_cast<LifeC*>(
             scene->getEntitybyId("Player")->getComponent("LifeC"));
-        // playerHealth->doDamage(damage);
+        /* // if player dies sleep method is called
+        if (playerHealth->doDamage(damage))
+            ;
+        // TO DO: call sleep() */
```

Comment:
I'm pretty sure you want to call doDamage, and that you wanted to comment was the sleep not being available yet.
```suggestion
        // if player dies sleep method is called
        if (playerHealth->doDamage(damage))
            ;
        // TODO(MiriamLeis): call `sleep()` when funcionality is available
```